modules/builtin/src/backend/variables/boolean.ts

```diff
@@ -1,7 +1,18 @@
-import { BoxedVarContructor, BoxedVariable } from 'botpress/sdk'
+import { BoxedVarContructor, BoxedVariable, FlowVariableConfig } from 'botpress/sdk'
 
 class BoxedBoolean implements BoxedVariable<boolean> {
   public static type = 'boolean'
```

Comment:
This static `type` property could probably be replaced by just using config.name